topics/climate/tutorials/argo_pangeo/tutorial.md

```diff
@@ -0,0 +1,281 @@
+---
+layout: tutorial_hands_on
+
+title: Analyse Argo data
+questions:
+- How to use Pangeo ecosystem to analyse ocean data?
+- How to process Argo data to visulise ocean temperature variations?
+objectives:
+- How to fetch Argo Data ?
+- How to get metadata with xarray netcdf tools ?
+- Learn to get temperature variations.
+- Learn to use an interactive tool to visualise Argo temperature variable on a interactive map.
+time_estimation: 1H
+key_points:
+- Argo data
+- Pangeo Ecosystem
+- Galaxy Earth System
+tags:
+  - earth-system
+  - ocean
+  - pangeo
+  - argo
+contributions:
+  authorship:
+    - Marie59
+  funding:
+    - fairease
+    - eurosciencegateway
+
+---
+
+The ocean is a key component of the Earth climate system. It thus needs a continuous real-time monitoring to help scientists better understand its dynamic and predict its evolution. All around the world, oceanographers have managed to join their efforts and set up a Global Ocean Observing System among which Argo is a key component.
+Argo is an international program that collects information from inside the ocean using a fleet of robotic instruments that drift with the ocean currents and move up and down between the surface and a mid-water level. 
+
+The data used in this tutorial are from the Argo gliders network. We are interested in the following variables: water temperature, latitude, longitude and time. Our main objective is to plot the water temperature with respect to time. For this we will be using the netCDF xarray tools available in the Galaxy Europe (or your favourite Galaxy Instance) server.
+
+
+These tools are part of the Pangeo ecosystem in which the next generation of open-source analysis tools for ocean, atmosphere and climate science can be developed, distributed, and sustained. These tools must be scalable in order to meet the current and future challenges of big data, and these solutions should leverage the existing expertise outside of the geoscience community.
+
+This tutorial is part of set of tutorials on the [Galaxy Earth System](https://earth-system.usegalaxy.eu/) supported by the [EOSC FAIR-EASE project](https://fairease.eu/).
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+> <details-title>Short introduction on how Galaxy works</details-title>
```

Comment:
do you want to add galaxy 101 short as a 'requirement' in the metadata? It could help indicate to students where to find the pre-requisite material.

```
requirements:
  - type: "internal"
    topic_name: introduction
    tutorials:
      - galaxy-intro-short
```